docs/source-pytorch/ecosystem/community_examples.rst

```diff
@@ -1,36 +0,0 @@
-:orphan:
-
-Community Examples
```

Comment:
not sure if this way or other but I think preserving link to some good community project is useful

Reply:
I think it's a good idea to have such a page, but we need to put in the work to make it up-to-date and include recent and relevant projects. Since this page is not visible and hidden, I'll remove it now to avoid the link checker getting blocked on links here (like it happened in the other pages)